kythe/go/util/riegeli/riegeli_test.go

```diff
@@ -93,6 +113,60 @@ func testReadWriteStrings(t *testing.T, opts *WriterOptions) {
 	}
 }
 
+func writeProtos(t *testing.T, opts *WriterOptions, n int) *bytes.Buffer {
+	var buf bytes.Buffer
+	wr := NewWriter(&buf, opts)
+
+	for i := 0; i < n; i++ {
+		if err := wr.PutProto(numToProto(i)); err != nil {
+			t.Fatalf("Error PutProto(%d): %v", i, err)
+		}
+	}
+	if err := wr.Close(); err != nil {
+		t.Fatalf("Close error: %v", err)
+	}
+	return &buf
+}
+
+func numToProto(i int) *rtpb.Complex {
```

Comment:
Please add a comment here explaining what this does, generally.

Reply:
👍